core/feconf.py

```
@@ -698,14 +684,20 @@ def get_empty_ratings() -> Dict[str, int]:
 MESSAGE_TYPE_SUGGESTION = 'suggestion'
 
 MODERATOR_ACTION_UNPUBLISH_EXPLORATION = 'unpublish_exploration'
-DEFAULT_SALUTATION_HTML_FN: Callable[[str], str] = (
+# Here the names are intentionally UPPER_CASE since these are module-level
+# constants, even though pylint treats lambda assignments as functions.
+DEFAULT_SALUTATION_HTML_FN: Callable[[str], str] = (  # pylint: disable=invalid-name
     lambda recipient_username: 'Hi %s,' % recipient_username
 )
```

Ruff (0.15.21)

[error] Do not assign a `lambda` expression, use a `def`

Rewrite `DEFAULT_SALUTATION_HTML_FN` as a `def`

(E731)